test/integration/user_can_view_past_orders_test.rb

```diff
@@ -22,7 +22,6 @@ class UserCanViewPastOrdersTest < ActionDispatch::IntegrationTest
     visit '/orders'
 
     assert page.has_content?('Your Orders')
-    assert page.has_content?('ordered')
-    assert page.has_content?('paid')
+    assert page.has_content?('Paid')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.